extensions/mssql/package.json

```diff
@@ -16,7 +16,8 @@
   ],
   "scripts": {
     "compile": "gulp compile-extension:mssql-client",
-    "update-grammar": "node ../../build/npm/update-grammar.js Microsoft/vscode-mssql syntaxes/SQL.plist ./syntaxes/sql.tmLanguage.json"
+    "update-grammar": "node ../../build/npm/update-grammar.js Microsoft/vscode-mssql syntaxes/SQL.plist ./syntaxes/sql.tmLanguage.json",
+    "test": "mocha -r ts-node/register src/test/**/*.test.ts"
```

Comment:
Not necessary anymore since we synced up on how tests are run

Reply:
removed.